ui/Rhino/AGS/dev/AGS_toolbar_assign_forces_cmd.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import division
+
+import scriptcontext as sc
+
+import compas_rhino
+
+import AGS_form_check_dof_cmd
+import AGS_form_assign_forces_cmd
+import AGS_form_update_from_qind_cmd
+
+
+__commandname__ = "AGS_toolbar_assign_forces"
+
+
+def RunCommand(is_interactive):
+
+    if 'AGS' not in sc.sticky:
+        compas_rhino.display_message('AGS has not been initialised yet.')
+        return
+
+    scene = sc.sticky['AGS']['scene']
+    if not scene:
+        return
+
+    if not scene.find_by_name('Form'):
+        compas_rhino.display_message("There is no FormDiagram in the scene.")
+        return
+
+    AGS_form_check_dof_cmd.RunCommand(True)
+    AGS_form_assign_forces_cmd.RunCommand(True)
+    AGS_form_update_from_qind_cmd.RunCommand(True)
```

Comment:
I would not make these commands all together... It is too implicit. 

For me, only  `AGS_form_assign_forces_cmd.RunCommand(True)` needs to happen here.

We have a button check DOF, where the user can check the right number of inds.

Reply:
again, i would just get rid of all the toolbar commands for now...

Reply:
Did you discuss that the use would be told how many ind edges they should select before assigning forces?
Shall i revert to before.. @ricardoavelino

Reply:
I just don't think we should invoke the same command again. One suggestion is:

- In the GetObjects command we can specify a limit number, maybe we can try that... so we don't allow more then DOF edges to be selected: https://developer.rhino3d.com/api/RhinoScriptSyntax/#selection-GetObjects
- Then I would just throw an error message if user selected too few or too many...
This would be enough for now. 

In a future PR, we could run `AGS_form_update_from_qind_cmd.RunCommand(True)` to check if the DOF were selected correctly and also if the value of the force assigned is possible, i.e. if that command works or not.

Reply:
assigning forces should just be about assigning forces...

Reply:
So let's only make AGS_form_assign_forces_cmd.RunCommand(True) happen here, indeed.